### PR TITLE
Add normal map unpacking formula to normal map description

### PR DESCRIPTION
**Pull request by stevenvergenz**
See #993 for discussion

### CODE REVIEW

**Comment by lexaknyazev** [comment]
Properties Reference will be generated from JSON-Schema files, so its updates should go there.

**Comment by pjcozzi** [comment]
@stevenvergenz could you please also add yourself to the ** Acknowledgments** section?

File: specification/2.0/README.md [hunk 1]
```diff
@@ -25,7 +26,7 @@ Khronos 3D Formats Working Group and Alumni
 * Ed Mackey, Individual Contributor
 * Don McCurdy, Google
 * Norbert Nopper, UX3D
-* Fabrice Robinet, Individual Contributor (Previous Editor and Incubator)
```
Comment (emackey) [inline specification/2.0/README.md]: What happened here?

Comment (stevenvergenz) [inline specification/2.0/README.md]: Hmm, I have no idea. I'll put him back.

**Comment by pjcozzi** [comment]
Thanks again @stevenvergenz!  Also note that I moved your attribution from the editors section to the acknowledgment section.

**Comment by stevenvergenz** [comment]
Perfect, thank you!

**Pull request merged**